assets/sass/v2/_select-authenticator.scss

```diff
@@ -6,6 +6,9 @@
 }
 
 .authenticator-enroll-list {
+  .authenticator-description {
+    width: 100%;
```

Comment:
This allows our usage of `flex` to stretch to the full width. By setting this to `100%`, it'll expand to the parent container's width.